apps/core/factories.py

```diff
@@ -49,3 +49,85 @@ class OmicsUnitFactory(DjangoModelFactory):
     class Meta:
         model = 'core.OmicsUnit'
         django_get_or_create = ('reference', 'strain')
+
+
+class PixelerFactory(DjangoModelFactory):
+
+    date_joined = Faker('date_object')
+    email = Faker('email')
+    first_name = Faker('first_name')
+    is_active = Faker('boolean')
+    is_staff = Faker('boolean')
+    is_superuser = Faker('boolean')
+    last_login = Faker('date_object')
+    last_name = Faker("last_name")
+    password = Faker("password")
+    username = Faker("user_name")
+
+    class Meta:
+        model = 'core.Pixeler'
+        django_get_or_create = ('username',)
+
+
+class OmicsAreaFactory(DjangoModelFactory):
+    description = Faker('text', max_nb_chars=300)
+    level = Faker('pyint')
+    lft = Faker('pyint')
+    name = Faker('word')
+    rght = Faker('pyint')
+    tree_id = Faker('pyint')
+
+    class Meta:
+        model = 'core.OmicsArea'
+        django_get_or_create = ('name',)
+
+
+class TagFactory(DjangoModelFactory):
```

Comment:
This whole factory will not work with the way you defined it: some fields are interdependent and your implementation is not consistent. Moreover, this factory is not required as you can define tags via a tagged model:

```python
analysis.tags = 'foo/bar/lol'
analysis.save()
```